packages/web/docs/src/app/feed.xml/route.ts

```diff
@@ -1,26 +1,62 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import RSS from 'rss';
 import { getChangelogs } from '../../components/product-updates';
 
+// Generate a TypeScript file with the latest changelogs
+const outputFilePath = path.resolve(
+  '..',
+  'app',
+  'src',
+  'components',
```

Comment:
why not just `../app/src/components..`?

Reply:
https://github.com/graphql-hive/console/pull/6089/commits/8d3efd22d56da7b060bfd3f48d490ceb6bb026a1 😁